fn.go

```diff
@@ -36,7 +36,7 @@ func timeElapsed(xr *resource.Composite, rsp *fnv1beta1.RunFunctionResponse, log
 		annotations["poll.fn.kndp.io/last-sent-time"] = strconv.Itoa(currentTime)
 		xr.Resource.SetAnnotations(annotations)
 		if err := response.SetDesiredCompositeResource(rsp, xr); err != nil {
-			logger.Info("Error setting desired composite resource", err)
+			logger.Info("error setting desired composite resource", err)
```

Comment:
@ca7alindev User logger.Error instead, and without error word in message.

Reply:
@evghen7, that means to use "log" go default package instead of logging package from crossplane because its interface does not implement Error.